lib/cereal/postgres/sql_schema.go

```diff
@@ -255,39 +258,101 @@ $$;
 -- Task Functions
 CREATE OR REPLACE FUNCTION cereal_enqueue_task(
     _workflow_instance_id BIGINT,
-    _try_remaining INT,
     _start_after TIMESTAMPTZ,
     _task_name TEXT,
     _parameters BYTEA)
 RETURNS VOID
 AS $$
-    INSERT INTO cereal_tasks(workflow_instance_id, try_remaining, start_after, task_name, parameters)
-        VALUES(_workflow_instance_id, _try_remaining, _start_after, _task_name, _parameters);
+    INSERT INTO cereal_tasks(workflow_instance_id, start_after, task_name, parameters)
+        VALUES(_workflow_instance_id, _start_after, _task_name, _parameters);
 $$ LANGUAGE SQL;
 
 CREATE OR REPLACE FUNCTION cereal_dequeue_task(_task_name TEXT)
 RETURNS TABLE(id BIGINT, workflow_instance_id BIGINT, parameters BYTEA)
 AS $$
-    UPDATE cereal_tasks t1 SET try_remaining = try_remaining - 1, updated_at = NOW()
-    WHERE t1.id = (
-        SELECT t2.id FROM cereal_tasks t2
-        WHERE t2.task_name = _task_name AND t2.try_remaining > 0 AND start_after < NOW()
-        ORDER BY t2.enqueued_at FOR UPDATE SKIP LOCKED LIMIT 1
-    ) RETURNING t1.id, t1.workflow_instance_id, t1.parameters
-$$ LANGUAGE SQL;
+DECLARE
+    r cereal_tasks%rowtype;
+BEGIN
+    FOR r IN
```

Comment:
Now that we are back to updating a single table, I'm not sure this conversion to a `FOR` loop make sense anymore.